src/v2/guide/team.md

```diff
@@ -225,15 +225,18 @@ order: 803
     'Annecy, France': [45.899247, 6.129384],
     'Alicante, Spain' : [38.346543, -0.483838],
     'Amsterdam, Netherlands': [4.895168, 52.370216],
+    'Atlanta, GA, USA': [33.7676338, -84.5606888],
```

Comment:
Are these the correct coordinates? It should be `33.749000, -84.388000` based on what I found on Google.